tests/chainer_tests/links_tests/model_tests/test_vision.py

```diff
@@ -0,0 +1,220 @@
+import unittest
+import warnings
+
+import numpy
+
+from chainer import cuda
+from chainer.links.model.vision import resnet
+from chainer.links.model.vision import vgg
+from chainer import testing
+from chainer.testing import attr
+from chainer.variable import Variable
+
+
+@unittest.skipUnless(resnet.available, 'Pillow is required')
+class TestResNet50Layers(unittest.TestCase):
+
+    def setUp(self):
+        self.link = resnet.ResNet50Layers(pretrained_model=None)
+
+        # Suppress warning that arises from zero division.
```

Comment:
You could use `with testing.NumpyError(divide='ignore'):` before the code that will cause zero division. It will call `numpy.seterr(divide='ignore')`.